vcd/resource_vcd_org_vdc_access_control_test.go

```diff
@@ -30,7 +31,7 @@ func TestAccVcdOrgVdcAccessControl(t *testing.T) {
 		"AccessControlName2": accessControlName + "2",
 		"UserName":           userName1,
 		"UserName2":          userName2,
-		"PasswordFile":       orgUserPasswordFile,
+		"Password":           "mei2Heehiu>Smei2Heehiu>S",
```

Comment:
Plain text password?

Reply:
The purpose of this was to make the test pass. Failure is noted in the description. As this is a test - we don't care much that it is plain text one. However - the fair point is that probably both cases should be tested. I am checking a bit more now.

Reply:
Ok. So I tried to overthink this. This test uses passwords just for creating prerequisite users (`vcd_org_user`). The main purpose of this test is to test `vcd_org_vdc_access_control` and not `vcd_org_user`. I have returned to a plain test password `CHANGE-ME` just to avoid overcomplicating the setup of this test. Using a `password_file` would make it a lot more complicated with binary tests and we don't want to skip them here.

Both `password_file` and `password` cases for ``vcd_org_user` are tested in `TestAccVcdOrgUser*` tests.